src/main/kotlin/minesweeper/MinesWeeperController.kt

```diff
@@ -23,17 +27,22 @@ class MinesWeeperController(
     private fun startGame(minesWeeperService: MinesWeeperService, board: Board) {
         resultView.printStartGame()
         while (!minesWeeperService.isFinishGame()) {
-            val (row, column) = inputView.inputOpenCell()
-            if (!minesWeeperService.openCells(row, column)) {
-                resultView.printLoseGame()
-                break
-            }
-
-            resultView.printBoard(board)
+            if (openCell(minesWeeperService, board)) break
```

Comment:
행위가 분리되었지만 while안에 if문이 있기에 block이 2 depth인건 동일한 것 같습니다 🤔 

```suggestion
            while(minesweeperService.playable(board)) {
                request = inputView.inputOpenCell()
                minesweeperService.process(board, request)
                resultView.printBoard(board)
            }
```

이런식으로도 진행이 가능할 것 같네요